src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -7,13 +7,12 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.PowerShell.EditorServices
 {
-    using System.Management.Automation;
-    using System.Management.Automation.Runspaces;
-    using System.Threading;
     /// <summary>
```

Comment:
I dunno, I catch Visual Studio doing it from time to time.  Seems like if one `using` ends up there then Visual Studio puts all future ones there!